### ISSUE TITLE
Spleen Sweeper

### ISSUE THREAD

**Issue by lolkaspy** [ValveSoftware/Dota2-Gameplay#22633]
### Description

I have a skill issue in this mode, there is not enough time to pass, please add 30-60 sec to the total initial time, or replace splee decoration as first reward. Ofc it's not a bug, but idk where i should to say about

**Comment by tomasbruckner** [comment]
I agree, the reason people pass is that they mostly take screenshot, pause and then solve it.

Also some people report that they solve all 5 puzzles but still don't get 35k points.
